### PR TITLE
[FEATURE] get attribute value

### PR DESCRIPTION
**Pull request by iariasleon**
```
                    SUMMARY:
-------------------------------------------------
  - components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature >> passed: 172, failed: 0, skipped: 18 and total: 190 with duration: 29.194 seconds.
-------------------------------------------------
  Date: Thursday, March 10, 2016 15:43:59
-------------------------------------------------
1 feature passed, 0 failed, 0 skipped
172 scenarios passed, 0 failed, 18 skipped
1197 steps passed, 0 failed, 132 skipped, 0 undefined
Took 0m29.194s
```

### CODE REVIEW

File: test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature [hunk 1]
```diff
@@ -0,0 +1,1265 @@
...
+    # These headers below are appended to previous headers in get request
+    And modify headers and keep previous values "true"
+      | parameter | value      |
+      | Accept    | text/plain |
+    When get an attribute value by ID "room_1" and attribute name "temperature_0" if it exists
+    Then verify that receive an "OK" http code
+    And verify that the attribute value by ID is returned
+
+  @date_type @BUG_1885 @ISSUE_1604 @skip
+  # this scenario is only to automate a bug.
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature]: "automate a bug"? or maybe "to automate the check for a bug"? ;)

Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/attributes_values/get_attribute_value/get_attribute_value.feature]: fixed in 46fa64e

**Comment by fgalan** [comment]
LGTM

**Comment by jcanonav** [comment]
LGTM

**Pull request merged**